server.py

```diff
@@ -809,7 +809,7 @@ def create_interface():
     # Launch the interface
     shared.gradio['interface'].queue()
     if shared.args.listen:
-        shared.gradio['interface'].launch(prevent_thread_lock=True, share=shared.args.share, server_name='0.0.0.0', server_port=shared.args.listen_port, inbrowser=shared.args.auto_launch, auth=auth)
+        shared.gradio['interface'].launch(prevent_thread_lock=True, share=shared.args.share, server_name=shared.args.listen_host if shared.args.listen_host else '0.0.0.0', server_port=shared.args.listen_port, inbrowser=shared.args.auto_launch, auth=auth)
```

Comment:
Minor Python syntax thing: should be able to just do `shared.args.listen_host or '0.0.0.0'` and it should work the same as this

Reply:
I've updated the PR with the new syntax.